controllers/users.controller.js

```diff
@@ -6,17 +7,20 @@ exports.login = async (req, res, next) => {
     const { name, email, picture } = profile;
 
     let user = await User.findOne({ email });
+    let isInitialLogin = false;
 
     if (!user) {
       user = await User.create({
         name,
         email,
         money: 5000,
-        image_url: picture,
+        imageUrl: picture,
       });
+
+      isInitialLogin = true;
```

Comment:
```suggestion
      isNewUser = true;
```
`isInitialLogin` 이라는 변수명보다 다른 변수명이 좋을 것 같습니다. 계정이 생성된 이후에는 `InitialLogin`이 `false`가 되어야 할 것 같은 느낌적인 느낌이 들어서... 뭔가 헷갈리네요.. `isNewUser`같은 걸로 하면 좀 덜 헷갈리지 않을까 싶어요!

Reply:
동의합니다.